### DISCUSSION TITLE
Application Error while moving around large field array fields

### DISCUSSION THREAD

**Discussion by RickRyan26** [react-hook-form/react-hook-form#5733] [Q&A]
![Screenshot_20210624-230912](https://user-images.githubusercontent.com/72894373/123379302-c4b9c980-d542-11eb-8512-d316dbb919b7.png)

Thought? Thank you.

**Comment by bluebill1049** [comment]
need more detailed error.

**Reply by RickRyan26** [reply]
You're an idol to me so it's really fun that you responded! But your brilliant code is solid. I believe the error lies in the fact that I have react-hot-parser running in too big of a loop through too many useWatches at once. If I merge the watches into one component and let the parser rerun less then this Googleless error will go away. I think the error lies in too much JS running at once and tripping on itself. If I can fix this RHF is going to make me so rich and I can't wait!

**Reply by bluebill1049** [reply]
no worries at all @RickRyan26 hope your project wins!
